config/configuration.toml.in

```diff
@@ -20,6 +20,17 @@ default_vcpus = -1
 #default_memory = @DEFMEMSZ@
 disable_block_device_use = @DEFDISABLEBLOCK@
 
+# Enable pre allocation of VM RAM, default false
+#enable_mem_prealloc = true
+#
+# Enable Realtime QEMU
+#enable_realtime = true
```

Comment:
Similarly, I think we should document why this isn't the default - can you document in the comments the implication of enabling this?

Reply:
We can make this the default post PnP to evaluate its impact.